### PR TITLE
Use YAPF instead of Black

### PR DESCRIPTION
**Pull request by kaapstorm**
Uses YAPF to clean up the code a bit. It's not perfect, but it's quick and it's better.

Context: https://github.com/dimagi/commcare-export/pull/205

### CODE REVIEW

**Review by snopoke — APPROVED** [review]
I didn't review all the changes in detail but I'm assuming it's all formatting according to the YAPF config. Please call out anything that doesn't fit into that assumption.

**Comment by kaapstorm** [comment]
That's correct. No functional changes in here, just formatting, and a few "%" and "format()" strings written as f-strings.

File: commcare_export/builtin_queries.py [hunk 1]
```diff
@@ -22,22 +23,27 @@ def mapped_source_field(self):
...
 
 def compile_query(columns, data_source, table_name):
-    source = Apply(Reference('api_data'), Literal(data_source),
-                   Reference('checkpoint_manager'))
-    part = excel_query.SheetParts(table_name, [c.name for c in columns], source,
-                                  List([c.mapped_source_field for c in columns]),
-                                  None)
+    source = Apply(
+        Reference('api_data'), Literal(data_source),
+        Reference('checkpoint_manager')
```
Comment (millerdev) [inline commcare_export/builtin_queries.py:35]: This one is a bit annoying. Wonder why it didn't do
```suggestion
        Reference('api_data'),
        Literal(data_source),
        Reference('checkpoint_manager')
```

File: commcare_export/builtin_queries.py [hunk 2]
```diff
@@ -22,22 +23,27 @@ def mapped_source_field(self):
...
+    )
+    part = excel_query.SheetParts(
+        table_name, [c.name for c in columns], source,
+        List([c.mapped_source_field for c in columns]), None
```
Comment (millerdev) [inline commcare_export/builtin_queries.py:39]: Same here. Per my aesthetic preferences
```
this_style(is_for, cramming, as_many, args,
           on, each, line, as_is, possible)

# and

this_other_better_style(
    puts_each,
    argument,
    on_a,
    separate,
    line,
)
```
When I see the second, I expect each line to be a separate argument, so what YAPF is doing here is a little unexpected.

I forget if YAPF has an option for that?

**Review by millerdev — APPROVED** [review]
Also didn't review every line, but it seems like a big improvement. Thanks!

File: commcare_export/builtin_queries.py (hunk 2 above)
Comment (kaapstorm) [reply commcare_export/builtin_queries.py]: Yes! Great suggestion. See https://github.com/dimagi/commcare-export/pull/207/commits/f7ff584d5eb5640f734fa016c1fade20feb18a18

You have to be a bit careful, cos that setting can mess with comprehensions:
```python
    {foo: bar for foo, bar in some_dictionary.items() if do_i_want_this_one(bar)}
```
becomes
```python
    {foo: bar for foo,
     bar in some_dictionary.items() if do_i_want_this_one(bar)}
```

But if you unpack into a tuple, it does a better job:
```python
    {foo: bar for (foo, bar) in some_dictionary.items() if do_i_want_this_one(bar)}
```
becomes
```python
    {foo: bar for (foo, bar) in some_dictionary.items()
     if do_i_want_this_one(bar)}
```

**Pull request merged**